src/views/community/edit-event.php

```diff
@@ -0,0 +1,113 @@
+<?php
```

Comment:
Do we have a way only to inject the pieces that we need for the reCaptcha? 

If we make changes to CE forms and this extension is enabled, then this template will be out of date and maybe buggy (assuming that it will take time to update the extension if we make changes in the plugin). This could be a problem for support.

Reply:
@juanfra I couldn't find a way to inject things that would be needed. However, it could be done if we made some slight modifications to the original template files.

As far as I remember:
1. The `<form>` needs an id.
2. The submit button needs an extra class. We could introduce a filter to be able to add that.
3. The submit button needs four parameters, which could be done with another filter.
4. We need to find a way to disable the `captcha.php` file (which is for v2) in case we're using v3. Again, a filter can do the trick. 

I know that's a lot of filters. :) That would be a fast way to have a simpler extension without template overrides.

In general it would be best to do all this in core - I'm sure you agree - so we wouldn't need an extension. I'd be happy to work on it as time allows.